src/Native/Runtime/startup.cpp

```diff
@@ -145,29 +150,103 @@ static void CheckForPalFallback()
 #endif // _DEBUG
 }
 
+// Should match the constants defined in the compiler in HardwareIntrinsicHelpers.cs
+enum XArchIntrinsicConstants
+{
+    XArchIntrinsicConstants_Aes = 0x0001,
+    XArchIntrinsicConstants_Pclmulqdq = 0x0002,
+    XArchIntrinsicConstants_Sse3 = 0x0004,
+    XArchIntrinsicConstants_Ssse3 = 0x0008,
+    XArchIntrinsicConstants_Sse41 = 0x0010,
+    XArchIntrinsicConstants_Sse42 = 0x0020,
+    XArchIntrinsicConstants_Popcnt = 0x0040,
+    XArchIntrinsicConstants_Lzcnt = 0x0080,
+};
+
 void DetectCPUFeatures()
 {
-#ifdef PROJECTN // @TODO: CORERT: DetectCPUFeatures
-
-#ifdef _X86_
-    // We depend on fxsave / fxrstor.  These were added to Pentium II and later, so they're pretty well guaranteed to be
```

Comment:
I've deleted this check without replacement - codegen assumes SSE/SSE2 support on x86 unconditionally anyway and fxsave was introduced around SSE1.